cmd/ddev/cmd/list.go

```diff
@@ -5,17 +5,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// continuous, if set, makes list continuously output
-var continuous bool
-
-// activeOnly, if set, shows only running projects
-var activeOnly bool
-
-// continuousSleepTime is time to sleep between reads with --continuous
-var continuousSleepTime = 1
-
-// wrapListTable allow that the text in the table of ddev list wraps instead of cutting it to fit the terminal width
-var wrapListTableText bool
+var ListCommandSettings = ddevapp.ListCommandSettings{}
```

Comment:
I don't think ListCommandSetings needs to be global does it?

Reply:
Done